reana_server/cli.py

```diff
@@ -207,3 +208,14 @@ def create_user(ctx, email, user_access_token, admin_access_token):
             click.style('User could not be created: \n{}'
                         .format(str(e)), fg='red'),
             err=True)
+
+
+@click.command('start_scheduler')
```

Comment:
Just a cosmetic, should we use `-` to separate words in the commands as we do in [`consume-job-queue`](https://github.com/reanahub/reana-cluster/blob/master/reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml#L48)? It would affect [r-cluster PR](https://github.com/reanahub/reana-cluster/pull/138/files#diff-553f301b92242889471252a2c1462906R46).

Reply:
Thanks for the link